dynamo/src/test/java/no/bibsys/db/ItemDriverTest.java

```diff
@@ -0,0 +1,76 @@
+package no.bibsys.db;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.document.Item;
+
+import no.bibsys.utils.IoUtils;
+
+
+public class ItemDriverTest extends LocalDynamoTest {
+
+
+    private ItemDriver itemDriver;
+    private TableDriver tableDriver;
+
+    @Override
+    @Before
+    public void init() {
+        super.init();
+        itemDriver = ItemDriver.create(new DynamoDB(localClient));
+        tableDriver = TableDriver.create(localClient, new DynamoDB(localClient));
+    }
+
+    @Test
+    public void addAndGetItem() throws IOException {
```

Comment:
tests should explain the expected behaviour. E.g. shouldThrowExceptionWhenAddingExistingId. 
Test if the add, update,etc behave well in corner cases,
Examples (more things should be checked):
add an existing item,
update an non existing item
delete an non existing item